src/django_peerctl/management/commands/peerctl_set_peer_session_type.py

```diff
@@ -0,0 +1,30 @@
+import re
+
+from fullctl.django.management.commands.base import CommandInterface
+
+from django_peerctl.models.peerctl import PeerSession, Port
+
+
+class Command(CommandInterface):
+    """
+    Sets the peer_session_type of a PeerSession based on the ports properties
+    """
+
+    def run(self, *args, **kwargs):
+        qset = PeerSession.objects.all()
+
+        for session in qset:
+            port = Port().first(id=session.port.id)
```

Comment:
@jason20c Its possible for a session to not have a  port, should skip if port is none or port.id is 0